src/openrct2/peep/Peep.h

```diff
@@ -776,9 +776,7 @@ struct Peep : SpriteBase
     void RemoveFromRide();
     void InsertNewThought(PeepThoughtType thought_type, uint8_t thought_arguments);
     void FormatActionTo(Formatter&) const;
-    void FormatActionTo(void* args) const;
-    void FormatNameTo(Formatter&) const;
-    size_t FormatNameTo(void* args) const;
+    size_t FormatNameTo(Formatter&) const;
```

Comment:
```suggestion
    void FormatNameTo(Formatter&) const;
```

No need to return anymore